backend/core/src/seeds/listings/listing-detroit-10145.ts

```diff
@@ -0,0 +1,133 @@
+import { ListingSeedType, PropertySeedType, UnitSeedType } from "./listings"
+import { ListingStatus } from "../../listings/types/listing-status-enum"
+import { CountyCode } from "../../shared/types/county-code"
+import { CSVFormattingType } from "../../csv/types/csv-formatting-type-enum"
+import { ApplicationMethodType } from "../../application-methods/types/application-method-type-enum"
+import { ListingDefaultSeed } from "./listing-default-seed"
+import { UnitCreateDto } from "../../units/dto/unit.dto"
+import { BaseEntity, DeepPartial } from "typeorm"
+import { Listing } from "../../listings/entities/listing.entity"
+import { UnitStatus } from "../../units/types/unit-status-enum"
+import { ApplicationMethod } from "../../application-methods/entities/application-method.entity"
+import assert from "assert"
+
+const mcvProperty: PropertySeedType = {
+  buildingAddress: {
+    city: "Detroit",
+    state: "MI",
+    street: "4701 Chrysler Drive",
+    zipCode: "48201",
+    latitude: 42.35923,
+    longitude: -83.054134,
+  },
+  buildingTotalUnits: 194,
+  neighborhood: "Forest Park",
+}
+
+const mcvUnits: Array<UnitSeedType> = []
+
+const oneBdrmUnit: UnitSeedType = {
+  numBedrooms: 1,
+  status: UnitStatus.occupied,
+}
+
+for (let i = 0; i < 28; i++) {
+  mcvUnits.push(oneBdrmUnit)
+}
+
+const twoBdrmUnit = {
+  numBedrooms: 2,
+  status: UnitStatus.occupied,
+}
+
+for (let i = 0; i < 142; i++) {
+  mcvUnits.push(twoBdrmUnit)
+}
+
+const threeBdrmUnit = {
+  numBedrooms: 3,
+  status: UnitStatus.occupied,
+}
+
+for (let i = 0; i < 24; i++) {
+  mcvUnits.push(threeBdrmUnit)
+}
+
+assert(mcvUnits.length === mcvProperty.buildingTotalUnits)
+
+const mcvListing: ListingSeedType = {
+  applicationDropOffAddress: null,
+  applicationMailingAddress: null,
+  countyCode: CountyCode.detroit,
+  CSVFormattingType: CSVFormattingType.basic,
+  disableUnitsAccordion: true,
+  displayWaitlistSize: false,
+  hrdId: "HRD10145",
+  leasingAgentName: "Janelle Henderson",
+  leasingAgentPhone: "313-831-1725",
+  managementCompany: "Associated Management Co",
+  managementWebsite: "associated-management.rentlinx.com/listings",
+  name: "Medical Center Village",
+  reservedCommunityDescription: "",
+  status: ListingStatus.active,
+}
```

Comment:
I'm not sure if "Affordability Mix" is a private field, but if not we could add `amiPercentageMin: null` and `amiPercentageMax: 60`

Reply:
Done!

Reply:
Do the same for the others? (the spreadsheet has this data for most listings, and it'll be an important part of filtering)

Reply:
I was basing these seeds off only what was in ArcGIS. I'm not sure how this corresponds to the `Affordable_at_xx_pct_AMI`. I'll leave it for now and we can fix it later.